cardano-cli/src/Cardano/CLI/EraBased/Commands/Genesis.hs

```diff
@@ -84,11 +84,13 @@ data GenesisCreateStakedCmdArgs = GenesisCreateStakedCmdArgs
   } deriving Show
 
 data GenesisCreateTestNetDataCmdArgs = GenesisCreateTestNetDataCmdArgs
-  { specShelley :: !(Maybe FilePath) -- ^ Path to the @genesis-shelley@ file to use. If unspecified, a default one will be used if omitted.
+  { specShelley :: !(Maybe FilePath) -- ^ Path to the @genesis-shelley@ file to use. If unspecified, a default one will be used.
+  , specAlonzo :: !(Maybe FilePath) -- ^ Path to the @genesis-alonzo@ file to use. If unspecified, a default one will be used.
+  , specConway :: !(Maybe FilePath) -- ^ Path to the @genesis-conway@ file to use. If unspecified, a default one will be used.
   , numGenesisKeys :: !Word -- ^ The number of genesis keys credentials to create and write to disk.
   , numPools :: !Word -- ^ The number of stake pools credentials to create and write to disk.
   , stakeDelegators :: !StakeDelegators -- ^ The number of delegators to pools to create.
-  , numDrepKeys :: !Word -- ^ The number of DRep keys to create. Right now they receive neither delegation nor are registrated. This will come later.
+  , numDRepKeys :: !DRepCredentials -- ^ The number of DRep keys to create. Right now they receive neither delegation nor are registrated. This will come later.
```

Comment:
Please fix the comment, which is outdated now.

Reply:
Thanks, well spotted